README.md

```diff
@@ -53,8 +61,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
-
+A parser for this grammar would likely be roughly O(nlogn), because the runtime grows in proportion to the number of tiles, but not exactly one-to-one - The parser will sometimes need to parse three nonterminals in place of one, such as in the rule S -> RSR.
```

Comment:
I think your were thinking about runtime, and not memory, although I suspect your estimate is low: I think the runtime will be at least O(n<sup>2</sup>).

The parser takes its input one token at a time. It doesn't know where the end is
until it gets there. So the mechanical parsing process is to get the next token,
push it on a stack, and then compare the rest of the input. As a result, the
stack can grow to at least n/2, so the memory consumption will be O(n)

# -3